### ISSUE TITLE
Please provide documentation for how to use and modify.

### ISSUE THREAD

**Issue by prithvirajdesigner** [software-opal/embroidery-rust#9]

**Comment by Fahad090NP** [comment]
You can try this https://github.com/Fahad090NP/ButaButi library, but it is under-development as of now, but better!

**Comment by Prithviraj639** [comment]
> You can try this https://github.com/Fahad090NP/ButaButi library, but it is under-development as of now, but better!

the link is not opening

**Comment by Fahad090NP** [comment]
Yes this link https://github.com/Fahad090NP/Embricad is working. This is an embroidery designs viewer.
